message_definitions/v1.0/ras_a.xml

```diff
@@ -16,6 +16,67 @@
         <param index="2" label="X Position in the camera frame" minValue="0" increment="1">Position given by the number of pixels in X direction of the camera frame. Origin on the bottom left corner of the camera frame.</param>
         <param index="3" label="Y Position in the camera frame" minValue="0" increment="1">Position given by the number of pixels in Y direction of the camera frame. Origin on the bottom left corner of the camera frame.</param>
       </entry>
+      <entry value="36" name="MAV_CMD_DO_FOLLOW_SYS_ID" hasLocation="false" isDestination="false">
```

Comment:
This already exists: [`MAV_CMD_DO_FOLLOW`](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_FOLLOW). Why exactly are we repeating it here?

Reply:
Good point @TSC21 !  I'm fine with removing that.  @thomas-watters-skydio and @jillh54 we're not using `MAV_CMD_DO_FOLLOW_SYS_ID` at all yet, are we?

Reply:
Nuno's point is that this **duplicates** [MAV_CMD_DO_FOLLOW](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_FOLLOW). Unless it differs you wouldn't bother creating a new message.